### ISSUE TITLE
翻訳の方法について

### ISSUE THREAD

**Issue by sotayamashita** [nodeschool-jp/translators#1]
翻訳の流れとして以下のような方法を考えたのですがどうでしょうか？

翻訳の流れ：
1. 翻訳するレポジトリをfork
2. git remote add upstream レポジトリ
3. `feature/japanese`ブランチ作成
4. `feature/japanese`をdefault branchにする
5.  nodeschoo-jp/レポジトリをfork
6.  nodeschoo-jp/レポジトリにPull Request

例、[`npm/how-to-npm`](https://github.com/npm/how-to-npm)の翻訳：
1. `https://github.com/npm/how-to-npm`をfork
2. `git remote add upstream https://github.com/npm/how-to-npm`
3. `git checkout -b feature/japanese`
4. `feature/japanese`をdefault branchにする
5. `https://github.com/nodeschool-jp/how-to-npm`をfork
6. `nodeschool-jp/how-to-npm` にプルリクエスト

**Comment by tai2** [comment]
LGTMです。

**Comment by kohei-takata** [comment]
LGTMです！

**Comment by tgfjt** [comment]
:+1:

**Comment by tai2** [comment]
@sotayamashita あ、そうだ。PR出すまではいいとして、レビューとマージは、どうしましょうかね。

**Comment by tai2** [comment]
あ、3のgit branch -b feature/japaneseってなってるのは、たぶん、git checkout -b feature/japanese ですかね。いちおう。

**Comment by sotayamashita** [comment]
> あ、そうだ。PR出すまではいいとして、レビューとマージは、どうしましょうかね。

@tai2 そうですね。自分はオープンソースにそこまでまだ関わったことがないのでわからないのですが、マージするかどうかはコラボレーターの半数がOKみたいな感じなんですかね？

**Comment by sotayamashita** [comment]
> あ、3のgit branch -b feature/japaneseってなってるのは、たぶん、git checkout -b feature/japanese ですかね。いちおう。

@tai2 ありがとうございます。修正いたしました

**Comment by sotayamashita** [comment]
wikiで翻訳の流れ等をwikiにまとめております。

**Comment by tai2** [comment]
@sotayamashita ぼくもオープンソースぜんぜん詳しくないです。仕事のやつだったら、チームメンバーが全員目を通したらOKとかだと思うんですが、こういうケースだとどうしたらいいのか、難しいですね。オープンソースでの共同作業に詳しい人の意見を待ちたい所です。

**Comment by tgfjt** [comment]
全員目を通すのは難しいと思うので、ある程度LGTMもらったらマージで良いのではないでしょうか。
誰も数日見てくれなかったらマージしちゃっても良いかもです。

翻訳文だけですし、おかしかったらまた直しましょう、でも良いと思います。

**Comment by sotayamashita** [comment]
@tgfjt そうですね。何人かオープンソースに関わった人に聞いてみたのですが翻訳だけでればそこまで重大なバグとかがないので翻訳したらどんどんマージして全体のレベルを上げていけばいいと思います

**Comment by tgfjt** [comment]
誰がやるか等何も話しあえていなかったのですが、時間も迫っているので、
how-to-npm-jp は、自分の方で merge と publish しちゃいました。
これでメンターの方々もインストールしやすいと思います。

もう少し、マージなど権限を広めても良いと思っています。
@sotayamashita

**Comment by tai2** [comment]
お、公開されてますねー。ありがとうございます。 :+1:

**Comment by sotayamashita** [comment]
@tgfjt 

確かに、 @tai2 さんなどは自分よりも活発に協力してくださっているのでマージなどを権限があったほうがストレスなく翻訳ができますね。権限をadminに修正致しました。
